.github/workflows/master.yaml

```diff
@@ -197,6 +197,7 @@
    "on": {
       "push": {
          "branches": [
+            "main",
```

Comment:
What does adding `main` fix? I thought this repo is only using `master` as the SoT branch